src/package-linker.js

```diff
@@ -436,7 +436,12 @@ export default class PackageLinker {
         topLevelDependencies,
         async ([dest, {pkg}]) => {
           if (pkg._reference && pkg._reference.location && pkg.bin && Object.keys(pkg.bin).length) {
-            const binLoc = path.join(this.config.lockfileFolder, this.config.getFolder(pkg));
+            let binLoc;
```

Comment:
Prefer `const`:
```js
const binLoc = this.config.modulesFolder ? this.config.modulesFolder : path.join(this.config.lockfileFolder, this.config.getFolder(pkg));
```

Reply:
done

Reply:
eslint is unhappy because `Line 439 exceeds the maximum line length of 120`. How should I write this line without `if` and `let` ?